### ISSUE TITLE
Addition: Paid Staff 252

### ISSUE THREAD

**Issue by BAAugspurger** [IMLS/public-libraries-survey#74]
Item 252: I was approached at state level with a concern of minimizing "Other Staff" who are highly qualified individuals that are not classified as Librarians.  What are your thoughts on separating "Other Staff" and adding a new line item "Other Staff-MS" to this section.  

Definition Suggestion: Other paid staff with at least a Master's Degree from certified programs of study from an accredited college.

Reasoning: Highly qualified employees from other fields can help tell the story of services provided within communities and services provided by the library system.  These same subject matter experts can help diversify opportunities and resources for patrons and stakeholders.

Thank you for consideration.   #2026  #Addition (Sorry, not sure how to add labels)

**Comment by mgolrick** [comment]
Louisiana has been asking this question for a very long time. We collect data in these three groups:

- All ALA/MLS degreed staff
- Other professional level staff
- All other paid staff

We add the first two to report: TOTAL staff with title of librarian

Here is our definition of "Other professional level staff"

> Staff who perform work that requires professional training and skill in the theoretical or scientific aspects of library work, as distinct from its mechanical or clerical aspects. Include the director without the MLS and others who hold the title of librarian, such as library associates.

**Comment by angelakfox** [comment]
Michael - I'm not entirely sure this is the same as what is being proposed. My understanding of the original suggestion was to add a question that very specifically asked about professionals (**as defined by personnel having an masters degree**), whereas it seems like the LA state-added question is about **people who do professional library work without a library degree.** The former seeks to count those with any sort of master's who are working in the library world, while the latter attempts to count those working as a professional librarian without a master's. My apologies to you and the original poster if I've misinterpreted these questions!

If I am in fact understanding the original proposal, I am reluctant to get behind it as I'm not fully sold on the need. Part of the reasoning provided was that "[h]ighly qualified employees from other fields can help tell the story of services provided within communities and services provided by the library system" - but simply counting them doesn't give us an idea of what those degrees are for and how they might impact/improve services provided by the library system. Do these people get to use their (degree) knowledge in any way in their library work (e.g., a person with a master's in history working in a genealogy department might make the case that their understanding of certain period helps them educate researchers coming in to their department), or is it more like a former colleague of mine who realized midway through life that no, he actually _didn't _ enjoy his banking job (for which his Master's of Finance had helped him secure) who turned around and spent the second half of his working life manning the reference desk and running a book group at his library. The argument that "this non-library degree provides added value" can still be made - but it's not nearly as obvious and perhaps not as helpful. And again, without knowing the exact degrees (and I'd rather not have a staff of 400 broken down into individual employee degrees!), I'm not sure how much you can extrapolate from the numbers beyond "sometimes highly educated people end up working for the library."

A couple of secondary concerns: how would the reporting library know what degrees their staff held without looking at past resumes or surveying their employees? I believe the number of "librarians w/ MLS" vs "librarians without" is determined by the vast majority of our AEs by looking at the position requirements, as opposed to the people filling it. For extremely large systems - those the most likely to have multiple people with MLS - asking them to determine the status of current employees seems an burdensome task with little potential payoff.

If I'm misunderstanding the proposal and it's really meant to capture **positions requiring a master's degree - other than that of librarian** (such as the master's that could be required for a financial officer), I can't help but think this only the largest systems would regularly require a (non-library) masters. If it only applies to a small percentage of US libraries, does it justify asking it on a national level?

**Comment by buzzySLO** [comment]
I agree with @angelakfox. I think this question would generate a lot of confusion. Oregon used to ask this question. We stopped because it wasn't relevant to most of our largely rural libraries and because libraries didn't seem to use the statistic. It seems like this question is trying to measure "highly qualified" staff other than librarians. However, master's degrees aren't always the standard for a lot of the other professionals who work at a libraries - e.g. accountants, HR, IT - as Louisiana's definition indicates.
